build.gradle

```
@@ -137,6 +138,10 @@ test {
   systemProperties['hudson.model.DownloadService.noSignatureCheck'] = 'true'
   ignoreFailures = ciBuild
   maxParallelForks = project.maxParallelForks
+  retry {
+    maxRetries = 3
```

```suggestion
    maxRetries = 1
```

Let's use one retry. I hope the tests are not that flaky.

---

Sure we can.